### DISCUSSION TITLE
Dangers of Running Scripts Straight From the App

### DISCUSSION THREAD

**Discussion by olumolu** [Jeffser/Alpaca#340] [General]
Is it safe for users script can do nefarious stuff.

**Comment by Jeffser** [comment]
Welp that's what the disclaimer is there for, there's one limitation though, scripts can't ask for sudo (root permissions), of course scripts can still mess up your home folder if you give it permission to do so, by default Alpaca is well sandboxed with Flatpak and Snaps and it shouldn't be possible to damage any data outside of the app itself
